### ISSUE TITLE
How to deploy to EKS Cluster if I will convert with Kompose do it's will be enough?

### ISSUE THREAD

**Issue by developer-jfr** [judge0/judge0#407]

**Comment by AayushbajajCAW** [comment]
Yes can canybody help with this i tried with kompose the containers were never created for api or worker does anyone have any leads
This is the state of pods.
![Screenshot from 2023-05-24 15-02-25](https://github.com/judge0/judge0/assets/118883463/3b7d2481-de8b-4437-a0c3-74547f820630)

**Comment by raviprakash007** [comment]
I have started working on Judge0 in k8 cluster

**Comment by SujithThirumalaisamy** [comment]
> I have started working on Judge0 in k8 cluster

Do you have any progress on this? I am trying to deploy it in a autoscale manner. Can help you if needed!
